receiver/aerospikereceiver/integration_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2022, OpenTelemetry Authors
```

Comment:
We should add a build constraint for the `integration` tag, so this only runs when that tag is specified. 

Ex: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/76d58ec42d82e6a51a366a2607cdefcb2e410c36/receiver/mysqlreceiver/integration_test.go#L15